.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Did you commit the visual code settings on intend? Will they bw user in the GitHub Codespaces then?

Reply:
Nope, this file can be removed.

Reply:
The settings will be use by the Codespaces editor. Kinda intended. :-)